### PR TITLE
redesign: [price-section] [desktop] [ui-only] buy, offer, list, transfer, change price, and not listed section

### LINKED ISSUE #4505: redesign: [price-section] [desktop] [ui-only] buy, offer, list, transfer, change price, and not listed section

### PR DESCRIPTION
**Pull request by preschian**
**Thank you for your contribution** to the [KodaDot NFT gallery](https://kodadot.xyz).

👇 \_ Let's make a quick check before the contribution.

## PR Type

- [ ] Bugfix
- [ ] Feature
- [ ] Refactoring
- [x] Redesign

## Context

- [x] Closes #4505
- [x] use bold for the active state in activity tabs
- [x] use `formatDistanceToNowStrict` to trim `about`

#### Before submitting pull request, please make sure:

- [x] My contribution builds **clean without any errors or warnings**
- [x] I've merged recent default branch -- **main** and I've no conflicts
- [x] I've tried to respect high code quality standards
- [x] I've didn't break any original functionality
- [x] I've posted a screenshot of demonstrated change in this PR

#### Optional

- [ ] I've tested it at </bsx/collection>
- [ ] I've tested PR on mobile
- [ ] I've written unit tests 🧪
- [ ] I've found edge cases

#### Had issue bounty label?

- [ ] Fill up your KSM address: [Payout](https://beta.kodadot.xyz/transfer/?target=<My_Kusama_Address_check_https://github.com/kodadot/nft-gallery/blob/main/CONTRIBUTING.md#creating-your-ksm-address>)

#### Community participation

- [x] [Are you at KodaDot Discord?](https://discord.gg/35hzy2dXXh)

## Screenshot 📸

- [x] My fix has changed **something** on UI; a screenshot is best to understand changes for others.

image

### CODE REVIEW

**Review by roiLeo — COMMENTED** [review]
- From User Experience, is there any way to cancel button sliding? (initial position)
Screenshot 2022-12-16 at 10-46-15 KodaDot - Kusama NFT Market Explorer

- Maybe rename `GalleryItemActionCta` to something more meaninglfull since it can be reused on other component

- I can't choose expiration date (maybe use radio button?)

Screenshot 2022-12-16 at 10-50-42 KodaDot - Kusama NFT Market Explorer

otherwise the rest of the code looks good to me

**Comment by exezbcz** [comment]
Nice job, it looks really good! 

One issue, we had chat in the parent issue about the responsive version. AnnndI have to rethink it because for example from 1024 to 768px does not makes sense to have this animation etc. I did not think about it that much. I'll attempt to come up with something better - it seems to be a popup that will transform into a modal on mobile.

I mean on desktop with full view it looks really good and i am conviced this was good solution, but looking at the ui in the device toolbar i think we have to change few things. 

This was my fault and i am sorry for that.

File: components/gallery/GalleryItemAction/GalleryItemActionSection.vue [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+const priceUsd = ref('0')
+
+watchEffect(async () => {
+  const ksmPrice = localStorage.getItem('KSM') || (await getKSMUSD())
+  const { symbol } = assets(tokenId.value)
+  const price = format(props.price || '0', decimals.value, '')
+  const chainSymbol = urlPrefix.value === 'rmrk' ? unit.value : symbol
+
+  priceChain.value = `${parseFloat(price).toFixed(2)} ${chainSymbol}`
+  priceUsd.value = `$${Math.round(Number(price) * Number(ksmPrice))}`
```
Comment (roiLeo) [inline components/gallery/GalleryItemAction/GalleryItemActionSection.vue:42]:
```suggestion
  priceUsd.value = Math.round(Number(price) * Number(ksmPrice))
```
Do you really need template literals her?

**Review by roiLeo — APPROVED** [review]
✅ code lgtm, we can continue with this

**Comment by yangwao** [comment]
> At least deliver that first for v1.0 and adjust a little bit for mobile based on the desktop version.

Yes, agree with this approach. 
We've had a quick update on how to save some space and make mobile adjustments. 
I guess we will come up with a good mobile solution and how to reuse components from mobile to desktop for example.

**Comment by yangwao** [comment]
> Screenshot 2022-12-16 at 10-50-42 KodaDot - Kusama NFT Market Explorer

Issue is with animation is too wide and if we account for future narrow devices, foldable, tablets we need to come up with something flexible and probably go more vertical space 

To continue, we will track it in a separate mobile gallery item issue.

**Comment by exezbcz** [comment]
Hi! nice job

feedback
- background of the card should be k-darkmode, not black
- is it also possible to make the stroke around the frame as in design?
image

rest i think would be fixed in #4444 , for example stretching the card to something like this, which isnt expected behavior
image

Thanks!

**Comment by yangwao** [comment]
I guess let's merge it and then we can look adding #4531 and then through feedback process to tune details?

**Comment by yangwao** [comment]
I'm moving this forward

**Pull request merged**